File: Nível 5/3241 - Ajude um Candidato a PhD!/main.lua
quantia = tonumber(io.read())

for i = 1, quantia do 
    texto = io.read()

    if texto == "P=NP" then 
        print("skipped")
    else
        a, b = texto:match("(%S+)%+(%S+)")
        a, b = tonumber(a), tonumber(b)
        calc = a + b 
        print(calc)
    end
end

File: Nível 5/3241 - Ajude um Candidato a PhD!/main.py
quantia = int(input("A: "))

for i in range(1, quantia + 1):
    texto = input("B: ")

    if texto == "P=NP":
        print("skipped")
    else:
        texto = texto.split("+")
        a = int(texto[0])
        b = int(texto[1])
        calc = a + b 
        print(calc)